src/plot_api/accessibility.js

```diff
@@ -0,0 +1,71 @@
+"use strict";
+import c2mChart from "chart2music";
+
+export function enable (gd) {
+
+    const c2mData = {};
+    const labels = [];
+    
+    const fullData = gd._fullData;
+
+    for(var i = 0; i < fullData.length; i++) {
+        var trace = fullData[i] ?? {};
+        var {type, x = [], y = [], name = i, text = []} = trace;
+        if(type === 'scatter') {
+            var traceData = [];
+            if ('y' in trace) {
```

Comment:
In scatter traces with `orientation: 'h'`, we should use `trace.x` instead of `trace.y`.